### DISCUSSION TITLE
Compact mode

### DISCUSSION THREAD

**Discussion by thejoabo** [thejoabo/virtualfisher-bot#23] [Ideas]
@yudhistiraindyka I just finish the new compact menu ( #15 ). What do you think about this new design ? Any suggestions or ideas about important things that should be in it ? I'm unsure if is really important show the config info at all times, I was thinking about a keybind that alternates between inventory, status and config info. The minimum size is: 70 x 18, that should be small enough, right ?

![image](https://user-images.githubusercontent.com/49594896/181829408-e13b0b61-93fa-46f7-9b8e-aff5d66a60f6.png)

**Comment by brodycritchlow** [comment]
It looks good! Has the important details, thanks for all the hard work on the project. Is the compact version released on the github?

**Reply by thejoabo** [reply]
Not yet, I'll release when i finish the update, I'm changing a lot of stuff (like compartmentalization of features, captcha detection methods, async logic to messages, cooldown ( #12 ), ...). The probable release date should be 07/31 (but no promises, testing things takes a lot of time lol).

**Reply by brodycritchlow** [reply]
Seems good! Thanks for the effort :D

**Comment by yudhistiraindyka** [comment]
Hello @thejoabo, that looks really great, I like the idea for keybinds to change between different pages.

The compact window size should be fine and I believe it would be more accessible for a lot of people.

**Reply by brodycritchlow** [reply]
Agreed on the keybinds part.

**Reply by thejoabo** [reply]
Ok, thanks for the feedback guys !

**Reply by thejoabo** [reply]
Update: compact mode added on v1.2.0, but the keybind is still under development.
